source/extensions/filters/http/adaptive_concurrency/adaptive_concurrency_filter.cc

```diff
@@ -0,0 +1,70 @@
+#include "extensions/filters/http/adaptive_concurrency/adaptive_concurrency_filter.h"
+
+#include <chrono>
+#include <cstdint>
+#include <string>
+#include <vector>
+
+#include "common/common/assert.h"
+
+#include "extensions/filters/http/adaptive_concurrency/concurrency_controller/concurrency_controller.h"
+#include "extensions/filters/http/well_known_names.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace AdaptiveConcurrency {
+
+AdaptiveConcurrencyFilterConfig::AdaptiveConcurrencyFilterConfig(
+    const envoy::config::filter::http::adaptive_concurrency::v2alpha::AdaptiveConcurrency&,
+    Runtime::Loader& runtime, std::string stats_prefix, Stats::Scope& scope,
+    TimeSource& time_source)
+    : runtime_(runtime), stats_prefix_(std::move(stats_prefix)), scope_(scope),
+      time_source_(time_source) {
+  // TODO (tonya11en): Remove these noop calls when stats/runtime values are
+  // implemented.
+  //
+  // Calling for test coverage.
```

Comment:
nit: I think you can remove these calls. They don't do anything useful nor provide actual coverage.

Reply:
I should have mentioned that these are also to make the build pass. Clang really seems to dislike unused members.

I'm going to make use of them in the next step, so it'll simplify the next patch if I keep them as members and perform these unnecessary calls.